tests/integration-tests/jobs/createJobDocker-test.js

```diff
@@ -76,7 +76,9 @@ describe('Create job specific docker tests', async function () {
                             duration: 1,
                             environment: 'test',
                             run_immediately: true,
-                            max_virtual_users: 100
+                            max_virtual_users: 100,
+                            proxy_url: 'http://proxy.com',
+                            debug: '*'
```

Comment:
add proxy_url and debug to expectedResult

Reply:
added